### DISCUSSION TITLE
Unable to upgrade organisation plan

### DISCUSSION THREAD

**Discussion by scott-david-walker** [community/community#63584] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I’ve just created an organisation but when I click the “upgrade” button for the team plan the page just hangs and eventually crashes.

Has anyone came across this and potentially a workaround?

Cheers

**Comment by bf6** [comment]
Same

**Comment by Peiwvy** [comment]
Same problem too

**Comment by kylecharlesbotha** [comment]
I figured out the issue, rather, I think I have. 

Before trying to head to the upgrade page or clicking '_upgrade_' 

First head to your organization's _**billing page** under settings_, _https://github.com/organizations/{your_org_here}/settings/billing_

Scroll all the way to the bottom and **_invite yourself as a billing manager_**, once that's done you should be able to hit upgrade and proceed without any issues. 

I do think it is an unhandled bug/scenario though, hence the page crashing.

**Reply by kylecharlesbotha** [reply]
@bf6 @scott-david-walker @Peiwvy
